templates/single-product/related.php

```diff
@@ -46,9 +46,9 @@
 
 if ( $products->have_posts() ) : ?>
 
-	<div class="related products">
+	<section class="related products">
 
-		<h2><?php _e( 'Related Products', 'woocommerce' ); ?></h2>
+		<h2><?php echo sprintf( __( 'Other %s...', 'woocommerce' ), $product->get_categories() ); ?></h2>
```

Comment:
@claudiosmweb 
I found... get the main category** of each returned product by `get_related()` and work from that. Possibly, output the main category of each of them as long as they're not same... ah ?

** the default one, the one that we see next to the product name in the breadcrumb